### ISSUE TITLE
Mode 10 silo

### ISSUE THREAD

**Issue by envikowaty** [Courseplay/courseplay#6730]
Hello everybody !

Small issue with the mode 10 hmm cant really know how to exmplain it. After 3 runs tractor hit the wall....

![Clipboard01](https://user-images.githubusercontent.com/69640077/105766484-18a39080-5f5a-11eb-9ce3-783e6e517c93.jpg)

https://gist.github.com/envikowaty/9e2ef58cf1de5c3d41cd04f1f7945192

**Comment by Tensuko** [comment]
Course Wrong, Stoppoint way to close to Bunker.....

**Comment by envikowaty** [comment]
Ok so stop outside the bunker and start inside ?

**Comment by Tensuko** [comment]
yes, start inside, the stop point with some distance, becouse that is where he gets the target for his row, it to close, it won't be able to drive in curve to target without hitting the wall.

**Comment by Misamangame** [comment]
And I don't think having blade on back and roller in front will help. By my experience leave just 1 tool.

**Comment by envikowaty** [comment]
Hm I will try then !


Regarding the tools: blade is in back and roller in front

**Comment by Tensuko** [comment]
only the back tool will be controlled, what you have on the front is just  there.

**Comment by envikowaty** [comment]
Ok then 👌 so looks like everything is working fine. Small issue with saving routes at the moment only.

**Comment by Tensuko** [comment]
Alright, then Mode10 is working for you now.
